app/policies/organizations/question_policy.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Organizations
+  class QuestionPolicy < ApplicationPolicy
```

Comment:
I think we should use the `class Organizations::` module syntax for consistency.